PubHub/PubHub.API/Domain/Entities/Operator.cs

```diff
@@ -8,9 +8,9 @@ public class Operator
         public int? AccountId { get; set; }
         public required string Name { get; set; }
         public required string Surname { get; set; }
-        
+
         #region Navs
-        public required Account Account { get; set; }
+        public Account? Account { get; set; } = null!;
```

Comment:
Same as [this comment](https://github.com/Mike-Mortensen-Portfolio/PubHub_H6_Final/pull/65/files#r1542444589).